index.html

```diff
@@ -513,29 +513,140 @@ <h3>Assert Claim</h3>
       </dl>
     </section>
     <section>
-      <h3>Verify Claim</h3>
+      <h3>Verify Credential</h3>
       <dl class="dl-horizontal">
         <dt>Requirement</dt>
         <dd>
-It MUST be possible for a <a>verifier</a> to verify that the credential is an
-authentic statement of an <a>issuer's</a> <a>claims</a> about the
-<a>subject</a>. The verifying <a>entity</a> must have the capability to
-connect the issuer’s identity to its credential identifier and the
-<a>subject's</a> identity to their identifier as indicated in the credential.
-The issuer’s verification information, such as its public key, must be
-discoverable from the credential record and verifiably linked to the issuer.
-It MUST be possible to do this in an automated fashion.
+          It MUST be possible for a <a>verifier</a> to verify that the
```

Comment:
Assuming informative section
```suggestion
          It is expected to be possible for a <a>verifier</a> to verify that the
```

Reply:
Since this is a use cases and requirements document, I think its appropriate to state the requirements for the specification using normative language. 

I'll add something earlier on to clarify that the statements in this document are not normative for VC-Data-Model, but do describe requirements of the specification, based on known use cases.